shared/lib/sentry.ts

```diff
@@ -0,0 +1,42 @@
+import type * as Sentry from '@sentry/browser';
+
+/**
+ * Captures an exception event and sends it to Sentry.
+ *
+ * @param exception The exception to capture.
+ * @param hint Optional additional data to attach to the Sentry event.
+ * @returns the id of the captured Sentry event, or `undefined` if Sentry is not initialized.
+ */
+export function captureException(
+  exception: unknown,
+  hint?: Parameters<(typeof Sentry)['captureException']>[1],
+): string | undefined {
+  if (!globalThis.sentry?.captureException) {
+    console.warn('Sentry not initialized');
```

Comment:
In webpack dev builds it is expected for `globalThis.sentry` to be `undefined`, so these logs will add some noise in those builds. Probably fine, just thought it was worth mentioning.

Reply:
Hmm. Interesting. Maybe best to leave it for visibility, but hopefully this doesn't get too noisy. I guess it's a temporary problem anyway, as we'll be getting this working for webpack builds before long hopefully.

Reply:
Sentry works in webpack builds, its just not bundled in development builds by default, only for build performance reasons. It is automatically enabled in production builds and can be enabled in any build by adding the `--sentry` flag.